fe1-web/parts/lao/organizer/__tests__/RollCallOpened.test.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import { useRoute } from '@react-navigation/core';
+import { act, fireEvent, render } from '@testing-library/react-native';
+import {
+  Hash, Lao, LaoState, PublicKey, Timestamp,
+} from 'model/objects';
+import STRINGS from 'res/strings';
+import keyPair from 'test_data/keypair.json';
+import { requestCloseRollCall as mockRequestCloseRollCall } from 'network/MessageApi';
+import { OpenedLaoStore } from 'store';
+// @ts-ignore
+import { fireScan as fakeQrReaderScan } from 'react-qr-reader';
+
+import RollCallOpened from '../RollCallOpened'; // Don't change the order, otherwise the tests will fail
+
+jest.mock('@react-navigation/core');
+
+export const mockPublicKey = new PublicKey(keyPair.publicKey);
+
+const org = mockPublicKey;
+const time = new Timestamp(1609455600).toString(); // 1st january 2021
```

Comment:
I'd suggest storing the number 1609455600 in a constant, since you use it multiple times.